Formula/osinfo-db.rb

```diff
@@ -0,0 +1,17 @@
+class OsinfoDb < Formula
+  desc "Osinfo database of operating systems for virtualization provisioning tools"
+  homepage "https://libosinfo.org/"
+  url "https://releases.pagure.org/libosinfo/osinfo-db-20210426.tar.xz"
+  sha256 "66c93b4a5b319b2ff7d40360d9e94486a9a45f2c0dd1c30d4b31de4f223070d0"
+  license "GPL-2.0-or-later"
+
+  depends_on "osinfo-db-tools" => :build
```

Comment:
```
==> osinfo-db-validate --system
dyld: Library not loaded: /opt/homebrew/opt/glib/lib/libgobject-2.0.0.dylib
  Referenced from: /opt/homebrew/bin/osinfo-db-validate
  Reason: image not found
```
Looks like this needs `glib` at runtime.

Reply:
I'm not sure why I thought `:build` dependencies would be installed when testing. Whoops! I'll push a fix up shortly :smile:

Reply:
This has been solved with the most recent push.